### PR TITLE
Minor bug fixes as commented from other group

### LINKED ISSUE #197: Change links in DG
Currently pointing to Ab3's links

### PR DESCRIPTION
**Pull request by yucheng11122017**
Closes #197

### CODE REVIEW

**Review by Justinhoejj — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -430,10 +430,10 @@ Action | Summary | Format, Examples
...
-**Tag** | Tags a contact | `tag INDEX [t/TAG]…​`<br> e.g., `tag 2 t/friend t/NUS`
-**Untag** | Untags a contact | `untag INDEX [t/TAG]…​`<br> e.g., `untag 2 t/colleague`
-**Find** | Locates contacts by name and tags (Results fulfill all search terms)| `find [c/] [n/NAME] [t/TAG]`<br> e.g., `find n/James t/friends`
-**FindAny** | Locates contacts by name and tags (Results fulfill at least one search term)| `findAny [c/] [n/NAME] [t/TAG]`<br> e.g., `findAny n/James t/NUS`
+**Tag** | Tags a contact | `tag INDEX t/TAG [t/MORE_TAGS]…​`<br> e.g., `tag 2 t/friend t/NUS`
+**Untag** | Untags a contact | `untag INDEX t/TAG [t/MORE_TAGS]…​`<br> e.g., `untag 2 t/colleague`
+**Find** | Locates contacts by name and tags (Results fulfill all search terms)| `find [c/] [n/NAME]... [t/TAG]...` <br> e.g., `find n/James t/friends` <br> At least one argument must be present
+**FindAny** | Locates contacts by name and tags (Results fulfill at least one search term)| `findAny [c/] [n/NAME]... [t/TAG]...`<br> e.g., `findAny n/James t/NUS` <br> At least one argument must be present
```
Comment (Jiwei99) [inline docs/UserGuide.md:436]: Might want to add help also. I think that's missing.

File: docs/tutorials/AddRemark.md [hunk 2]
```diff
@@ -295,7 +295,7 @@ While the changes to code may be minimal, the test data will have to be updated
 
 <div markdown="span" class="alert alert-warning">
 
-:exclamation: You must delete AddressBook’s storage file located at `/data/addressbook.json` before running it! Not doing so will cause AddressBook to default to an empty address book!
+:exclamation: You must delete AddressBook’s storage file located at `/data/CONNECTIONS.json` before running it! Not doing so will cause AddressBook to default to an empty address book!
```
Comment (Jiwei99) [inline docs/tutorials/AddRemark.md:298]: Should change `AddressBook` to `CONNECTIONS`

**Review by Jiwei99 — APPROVED** [review]
Some comments

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -902,14 +884,13 @@ Step 3. CONNECTIONS will not display the first contact's invalid `Tag` and will
...
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 contacts without a noticeable sluggishness in performance for typical usage.
+2.  Should be able to hold up to 100 contacts without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 
 ### Glossary
 * **CONNECTIONS**: The name of our product
 * **Entry**: An item written or printed in a diary, list, account book, or reference book.
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
```
[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:893]: Apple trash either way HAHA

**Pull request merged**